### PR TITLE
Release 0.0.1

### PR DESCRIPTION
**Pull request by gregory-akins**
First release of @Madie/models

### CODE REVIEW

File: src/Measure.ts [hunk 1]
```diff
@@ -18,10 +26,13 @@ export default interface Measure {
...
+  measureScoring: MeasureScoring | "";
   cql: string;
   createdAt: string;
   createdBy: string;
   lastModifiedAt: string;
   lastModifiedBy: string;
+  model: Model | "";
   measureMetaData?: MeasureMetadata;
+  groups?: Array<Group>;
 }
```
Comment (jkotanchik-SB) [inline src/Measure.ts:38]:
```suggestion
  groups?: Array<Group>;
  elmJson?: string;
}
```
Until we have a

File: src/MeasureScoring.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+export type MeasureScoringKeys = keyof typeof MeasureScoring;
+
+/*
+Leaving this here because I don't completely understand what it was for and whether it's needed 
+
+Joe Said:
+
+  Joe Kotanchik:taco:  12:29 PM
```
Comment (jkotanchik-SB) [inline src/MeasureScoring.ts:15]: 🌮

File: .gitignore [hunk 3]
```diff
@@ -1,4 +1,6 @@
+coverage/
 dist/
 src/**/*.js
 src/**/*.d.ts
-*~
\ No newline at end of file
+*~
+node_modules/
```
Comment (adongare) [inline .gitignore:6]: looks like this was added later. node modules are still in commit

Comment (adongare) [reply .gitignore:6]: I think it would be good if we add `.npmignore` to ignore tests going into npm package

Comment (gregory-akins) [reply .gitignore:6]: Added .npmignore with **/__tests__/**/*

**Review by adongare — APPROVED** [review]

**Pull request merged**